### PR TITLE
Various edits to usage.md

### LINKED ISSUE #42: Add link to FAQ on MacOS Error to Usage Instructions
- [ ] Add a link to the FAQ on getting past "apple cannot check for malware" in the MacOS usage page.

### PR DESCRIPTION
**Pull request by CodeShady**
**Hey!**

I saw your issue when searching through the "issues" tab. I thought I could help out here!
Hopefully this is what you're looking for. If you need any more help, just let me know. 👍

## Changes:
- Added fancy icons to make the `usage.md` file look more professional, and clean.
- Added small help section for "Apple can’t check app for malicious software". Just so you know, **I did not add a link to the original FAQ website**. Instead, I moved the useful information from the site, to the `usage.md` file for a _swift_ and easy experience for any potential user's with this issue.
- Fixed various grammar problems. This was completely optional, but it makes your `usage.md` file look more professional.

### Linked Issues:
Closes #42 

### Category
- [ ] This is a Bugfix
- [ ] This is a Feature Addition
- [ ] This is a CI Change
- [x] This is a Documentation-Only Change (do not check any of the above boxes if this box is checked)
### Breaking/Non Breaking
- [ ] This is a Breaking Change
- [x] This is a Non-Breaking Change

### CODE REVIEW

**Comment by Morpheus636** [comment]
Hi! Thanks for your contribution, your changes look great and I really appreciate them.

Just a couple of things:
1. I've gone ahead and changed your selected Change Type to Documentation Only
2. I noticed you did not fill out the copyright assignment. Unfortuantely, for legal/copyright reasons, I need a copyright assignment from each contributor in the event that I need to enforce the license. If you have any concerns or questions about it, you can ask here or in an email to morpheus636@morpheus636.com

Thanks!

**Review by Morpheus636 — APPROVED** [review]

**Comment by CodeShady** [comment]
Okay, I tweaked a few other things and added a few more too.
I'm happy to fill out the copyright assignment. Could you inform me on how I'm supposed to do that?

**Review by Morpheus636 — APPROVED** [review]

File: docs/README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
+<img src="https://raw.githubusercontent.com/CodeShady/redownload/main/logo.png" />
```
Comment (Morpheus636) [inline docs/README.md:1]: Unless I decide to have a full-on logo made for the project (which is really not on the radar right now), I'd rather not put one in the README.

File: docs/README.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+
 # Redownload
-A python based utility for downloading mp3 files from relisten.
+<a href="https://discord.morpheus636.com"><img src="https://img.shields.io/badge/Discord-%237289DA.svg?style=for-the-badge&logo=discord&logoColor=white" /></a><a href="https://github.com/Morpheus636/redownload"><img src="https://img.shields.io/badge/github-%23121011.svg?style=for-the-badge&logo=github&logoColor=white" /></a>
```
Comment (Morpheus636) [inline docs/README.md:4]: These badges are really cool!

File: docs/README.md [hunk 3]
```diff
@@ -21,4 +26,8 @@ To ask a question or get support, you can join my [Discord Server](https://disco
 [Forums](https://forums.morpheus636.com)
 
 ## Contributing
-To contribute to this project, view the Contribution Docs found in [CONTRIBUTING.md](CONTRIBUTING.md)
\ No newline at end of file
+To contribute to this project, view the Contribution Docs found in [CONTRIBUTING.md](CONTRIBUTING.md)
+
+---
+
```
Comment (Morpheus636) [inline docs/README.md:32]: Put in a credits subhead. Probably a H2 or H3.

**Review by Morpheus636 — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 3 above)
Comment (Morpheus636) [reply docs/README.md:30]: Oh sorry I meant above where the actual credits are, put a subhead that says
## Credits

**Comment by Morpheus636** [comment]
> I'm happy to fill out the copyright assignment. Could you inform me on how I'm supposed to do that?

Certainly! Just fill out the form at https://copyright.morpheus636.com

**Comment by Morpheus636** [comment]
Thanks for filling that out! Just make the one change still pending and I'll go head and merge this.

**Comment by CodeShady** [comment]
Alright. I believe that's how you'd like it.

**Review by Morpheus636 — APPROVED** [review]
Looks good!

**Pull request merged**